rotkehlchen/tests/unit/decoders/test_eigenlayer.py

```diff
@@ -36,14 +38,26 @@ def test_deposit_token(database, ethereum_inquirer, ethereum_accounts):
         location_label=ethereum_accounts[0],
         notes=f'Burned {gas_amount} ETH for gas',
         counterparty=CPT_GAS,
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=112,
+        timestamp=timestamp,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.APPROVE,
```

Comment:
Explained here https://github.com/rotki/rotki/pull/7538/commits/6ca040fd2c2d8e7c50f2bbbb718ffb2f53b6f491 why it was missing